client/src/components/EmailFrontend/EmailFrontend.stories.js

```diff
@@ -0,0 +1,85 @@
+import fetchMock from "fetch-mock";
+import React from "react";
+
+import { EmailFrontend } from "./EmailFrontend";
+
+export default {
+  title: "EmailFrontend",
+  component: EmailFrontend,
+  argTypes: {
+    loading: {
+      defaultValue: true,
+    },
+    include_privacy: {
+      defaultValue: true,
+    },
+  },
+};
+
+const Template = (args) => {
+  const template = {
+    meta: {
+      subject_template:
+        "Report a problem: [${issue_type}], ${lang}, ${route}, ${sha}, ${client_id}",
+    },
+    issue_type: {
+      required: true,
+      value_type: "enums",
+      enum_values: {
+        bug: {
+          en: "Something is broken",
+          fr: "Quelque chose ne fonctionne pas",
+        },
+        typo: {
+          en: "It has a spelling or grammar mistake",
+          fr: "II y a une erreur d'orthographe ou de grammaire",
+        },
+        inaccurate: {
+          en: "The information is wrong",
+          fr: "L'information est erronée",
+        },
+        outdated: {
+          en: "The information is outdated",
+          fr: "L'information n'est plus à jour",
+        },
+        navigation: {
+          en: "I can't find what I'm looking for",
+          fr: "Je ne trouve pas ce que je cherche",
+        },
+        other: {
+          en: "Other",
+          fr: "Autre",
+        },
+      },
+      form_type: "checkbox",
+      form_label: {
+        en: "Select all that apply:",
+        fr: "Sélectionner toutes les cases qui s'appliquent :",
+      },
+    },
+    issue_details: {
+      required: true,
+      value_type: "string",
+      form_type: "textarea",
+      form_label: { en: "Details (required)", fr: "Détails (obligatoire)" },
+    },
+    sha: { required: true, value_type: "string", form_type: false },
+    route: { required: true, value_type: "string", form_type: false },
+    lang: { required: true, value_type: "string", form_type: false },
+    app_version: { required: true, value_type: "string", form_type: false },
+    client_id: { required: true, value_type: "string", form_type: false },
+    additional: { required: false, value_type: "json", form_type: false },
+  };
+  fetchMock.mock("*", template);
```

Comment:
Look out, `fetchMock` is global/persistent!

Should use a more specific than the wild card. Skimming the `fetchMock` docs, might also want to either use `.getOnce`, clean up all mocks during unmount, or both.